myuw/templates/handlebars/banner/profile.html

```diff
@@ -37,24 +37,33 @@ <h3 class="sr-only">Official name and student ID number</h3>
 
             {{#with password}}
             <div class="profile-passwords row">
-                <h3 class="sr-only">Password Information</h3>
-                {{#if has_active_med_pw}}
-                <div class="col-md-6 uwmedicine-password">
-                    <h4>UW Medicine Account</h4>
-                    {{#if med_pw_expired }}
-                        <span class="password-alert">Password expired on {{formatDateAsDate expires_med}}. <a href="https://services.uwmedicine.org/passwordportal/login.htm">Change your password to regain access.</a></span>
```

Comment:
Our current guideline for links in MyUW is that we only open things in a new window if the back button doesn't reliably get a user back to MyUW. That is, we assume any departure from MyUW is the workflow people are taking and MyUW usage is secondary. That may or may not be a good assumption for instructors; but maybe it is for staff?

Also, I've had on my mind to revisit this guideline based on a11y best practices.